cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -76,6 +78,33 @@ def new_snapshot(test_vm, has_name=True, memory=False):
 
 @pytest.mark.uncollectif(lambda provider:
     not provider.one_of(RHEVMProvider, VMwareProvider) or
+    (provider.one_of(RHEVMProvider) and provider.version < 4) or
+    current_version() < '5.8')
+def test_memory_checkbox(small_test_vm, provider, soft_assert):
+    # Make sure the VM is powered on
+    small_test_vm.power_control_from_cfme(option=small_test_vm.POWER_ON, cancel=False)
+    # Try to create snapshot with memory on powered on VM
+    has_name = not provider.one_of(RHEVMProvider)
+    snapshot1 = new_snapshot(small_test_vm, has_name=has_name, memory=True)
+    snapshot1.create()
+    assert snapshot1.exists
+    # Power off the VM
+    small_test_vm.power_control_from_cfme(option=small_test_vm.POWER_OFF, cancel=False)
+    small_test_vm.wait_for_vm_state_change(desired_state=small_test_vm.STATE_OFF)
+    soft_assert(small_test_vm.provider.mgmt.is_vm_stopped(small_test_vm.name), "VM is not stopped!")
+    # Try to create snapshot with memory on powered off VM
+    snapshot2 = new_snapshot(small_test_vm, has_name=has_name, memory=True)
+    try:
+        snapshot2.create(test_memory=True)
```

Comment:
`force_check_memory=True`

Reply:
My apologies, completely forgot about this one.